### ISSUE TITLE
Why is  args.vocab_size passed in sequence_loss_by_example?

### ISSUE THREAD

**Issue by chiphuyen** [hunkim/word-rnn-tensorflow#52]
I looked up the documentation for sequence_loss_by_example and it doesn't seem to be taking vocab_size as argument. I'd really appreciate it if you could help me understand what this argument is doing. Thanks a lot!
` loss = seq2seq.sequence_loss_by_example([self.logits],
                [tf.reshape(self.targets, [-1])],
                [tf.ones([args.batch_size * args.seq_length])],
                args.vocab_size)`

**Comment by hksonngan** [comment]
tensorflow version < 1.0 have arg num_decoder_symbols: Integer, number of decoder symbols (output classes).
